RealmSwift/LinkingObjects.swift

```diff
@@ -274,7 +274,8 @@ public final class LinkingObjects<T: Object>: LinkingObjectsBase {
        is empty.
      */
     public func minimumValue<U: MinMaxType>(ofProperty property: String) -> U? {
-        return rlmResults.min(ofProperty: property) as! U?
+        guard let value = rlmResults.min(ofProperty: property) else { return nil }
```

Comment:
I don't feel strongly about this though, so I'll switch to the `map` pattern since higher order functions generally are more readable.